src/oss/langgraph/add-memory.mdx

```diff
@@ -2141,3 +2141,12 @@ await checkpointer.deleteThread(threadId);
 
 **LangMem** is a LangChain-maintained library that offers tools for managing long-term memories in your agent. See the [LangMem documentation](https://langchain-ai.github.io/langmem/) for usage examples.
 :::
+
+## Database management
+
+If you are using Postgres or Redis to store short and/or long-term memory, you will need to call the LangGraph-provided `saver` or `store`'s `setup` method before you can use it with your database.
```

Comment:
This applies to any checkpointer or store really - any DB-backed persistence implementation would have migrations.

The convention is to define a setup() method to run the required migrations for that instance, but the user should check with their specific implementation of the BaseCheckpointSaver or BaseStore to confirm.

We'd recommend running migration as a dedicated deployment step. Or you can ensure they're run as a part of server startup.